core/server/src/test/java/alluxio/master/AlluxioMasterRestServiceHandlerTest.java

```diff
@@ -0,0 +1,245 @@
+    when(mMaster.getMasterAddress()).thenReturn(new InetSocketAddress("localhost", 8080));
+    when(mContext.getAttribute(MasterUIWebServer.ALLUXIO_MASTER_SERVLET_RESOURCE_KEY)).thenReturn(
+        mMaster);
+    mHandler = new AlluxioMasterRestServiceHandler(mContext);
+  }
+
+  @Test
+  public void getConfiguration() {
+    Response response = mHandler.getConfiguration();
+    assertNotNull("Response must be not null!", response);
+    assertNotNull("Response must have a entry!", response.getEntity());
+    assertTrue("Entry must be a SortedMap!", (response.getEntity() instanceof SortedMap));
+    SortedMap<String, String> entry = (SortedMap<String, String>) response.getEntity();
+    assertFalse("Properties Map must be not empty!", (entry.isEmpty()));
+  }
+
+  @Test
+  public void getRpcAddress() {
+    Response response = mHandler.getRpcAddress();
+    assertNotNull("Response must be not null!", response);
+    assertNotNull("Response must have a entry!", response.getEntity());
+    assertEquals("Entry must be a String!", String.class, response.getEntity().getClass());
+    String entry = (String) response.getEntity();
+    assertFalse("Properties Map must be not empty!", (entry.isEmpty()));
+  }
+
+  @Test
+  public void getMetrics() {
+    final int FILES_PINNED_TEST_VALUE = 100;
+    String filesPinnedProperty =
+        MetricsSystem.getMasterMetricName(FileSystemMaster.Metrics.FILES_PINNED);
+    Gauge<Integer> filesPinnedGauge = new Gauge<Integer>() {
+      @Override
+      public Integer getValue() {
+        return FILES_PINNED_TEST_VALUE;
+      }
+    };
+    MetricSet mockMetricsSet = mock(MetricSet.class);
+    Map<String, Metric> map = new HashMap<>();
+    map.put(filesPinnedProperty, filesPinnedGauge);
+
+    when(mockMetricsSet.getMetrics()).thenReturn(map);
+    MetricsSystem.METRIC_REGISTRY.registerAll(mockMetricsSet);
+
+    Response response = mHandler.getMetrics();
+    assertNotNull("Response must be not null!", response);
+    assertNotNull("Response must have a entry!", response.getEntity());
+    assertTrue("Entry must be a SortedMap!", (response.getEntity() instanceof SortedMap));
+    SortedMap<String, Long> entry = (SortedMap<String, Long>) response.getEntity();
+    assertFalse("Properties Map must be not empty!", (entry.isEmpty()));
+    assertTrue("Map must contain key " + filesPinnedProperty + "!",
+        entry.containsKey(filesPinnedProperty));
+    assertEquals(FILES_PINNED_TEST_VALUE, entry.get(filesPinnedProperty).longValue());
+  }
+
+  @Test
+  public void getStartTimeMs() {
+    Response response = mHandler.getStartTimeMs();
+    assertNotNull("Response must be not null!", response);
+    assertNotNull("Response must have a entry!", response.getEntity());
+    assertEquals("Entry must be a Long!", Long.class, response.getEntity().getClass());
+    Long entry = (Long) response.getEntity();
+    assertEquals(0L, entry.longValue());
+
+    when(mMaster.getStartTimeMs()).thenReturn(100L);
```

Comment:
Could we mock this at the start of the function and only test `getStartTimeMs` once?